writer.go

```diff
@@ -1363,6 +1369,9 @@ func (c *writerColumn) writeDataPage(page Page) (int64, error) {
 	}
 
 	uncompressedPageSize := buf.size()
+	if uncompressedPageSize > maxUncompressedPageSize {
+		return 0, fmt.Errorf("page size must not be larger than %d but was %d", maxUncompressedPageSize, uncompressedPageSize)
```

Comment:
Maybe we can shorten the error message a bit, what do you think about this?
```suggestion
		return 0, fmt.Errorf("page size limit exceeded: %d>%d", uncompressedPageSize, maxUncompressedPageSize)
```